x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -154,13 +164,17 @@ export const MisconfigurationFindingsDetailsTable = memo(
         <EuiPanel hasShadow={false}>
           <EuiLink
             onClick={() => {
-              navToFindingsByName(queryName, fieldName);
+              uiMetricService.trackUiMetric(
+                METRIC_TYPE.CLICK,
+                NAV_TO_FINDINGS_BY_HOST_NAME_FRPOM_ENTITY_FLYOUT
+              );
+              navToFindingsByHostName(queryName);
             }}
           >
             {i18n.translate(
               'xpack.securitySolution.flyout.left.insights.misconfigurations.tableTitle',
               {
-                defaultMessage: 'Misconfigurations ',
+                defaultMessage: 'Misconfigurations',
```

Comment:
Please attach of a screenshot after this change, I would assume it will remove the space between the table header and the icon attached to it.

Reply:
I didn't intend to push these changes. I revert them now.